### PR TITLE
2.452.1

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

**Comment by MarkEWaite** [comment]
I don't think that we want this change merged to the master branch.  We want the stable-2.452 branch to contain these changes but not the master branch.  The master branch builds the weekly release as far as I understand it.

**Comment by NotMyFault** [comment]
> I don't think that we want this change merged to the master branch. We want the stable-2.452 branch to contain these changes but not the master branch. The master branch builds the weekly release as far as I understand it.

GH doing GH things again, oops

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
Needs to target correct destination branch

**Comment by NotMyFault** [comment]
> Needs to target correct destination branch

But that has already been fixed?

File: Jenkinsfile.d/core/package [hunk 1]
```diff
@@ -83,7 +83,7 @@ pipeline {
     GPG_FILE                  = 'jenkins-release.gpg'
     GPG_PASSPHRASE            = credentials('release-gpg-passphrase-2023')
     PACKAGING_GIT_REPOSITORY  = 'git@github.com:jenkinsci/packaging.git'
-    PACKAGING_GIT_BRANCH      = 'master'
+    PACKAGING_GIT_BRANCH      = 'stable-2.452.1'
```
Comment (MarkEWaite) [inline Jenkinsfile.d/core/package:86]:
```suggestion
    PACKAGING_GIT_BRANCH      = 'stable-2.452'
```

File: profile.d/stable [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # WARNING: Any variables defined here, override those defined from the Jenkinsfile
 #
 #
-RELEASE_GIT_BRANCH=master
+RELEASE_GIT_BRANCH=stable-2.452.1
```
Comment (MarkEWaite) [inline profile.d/stable:5]:
```suggestion
RELEASE_GIT_BRANCH=stable-2.452
```

**Review by MarkEWaite — CHANGES_REQUESTED** [review]
Sorry that I missed the change of destination branch.  You're correct.  Thanks for doing that.

I believe that the branch name is `stable-2.452` rather than `stable-2.452.1`

**Review by MarkEWaite — APPROVED** [review]
Thanks!

**Comment by NotMyFault** [comment]
We should really look into automating these tasks 👀

**Pull request merged**